### DISCUSSION TITLE
Create a package locally

### DISCUSSION THREAD

**Discussion by loictlpk** [Homebrew/discussions#1966] [Everyday usage]
Hello,

i write this article because i'm stuck on a step.
I wonder if it is possible to create a package on my pc without using github.
basically brew will have to create a package on my pc and then I have to use brew to install this package.
Is it possible ?
So, i've a pdk directory (i use puppet) and i edited his files. Now, i would like to deploy locally (on my personnal computer without github) my puppet code in the pdk directory with homebrew.

Thank community.

**Comment by SMillerDev** [comment]
Please don't make a new discussion because the old one didn't get answered.
